### DISCUSSION TITLE
Transmitting and Receiving a File via FSK

### DISCUSSION THREAD

**Discussion by ghost** [duggabe/gr-control#18] [Q&A] [answered]
Hi,

I'm trying to transmit and receive a file between two Lime Minis 2.0 using the [FSK example](https://wiki.gnuradio.org/index.php/Simulation_example:_FSK) on GNU Radio. Transmitting side seems to be functioning according to spectrum analyzer activity, but I am not getting data on the receiving end.

Here's the flow graphs:

![TX_LimeSDR](https://github.com/user-attachments/assets/688c53b3-2815-47f4-9660-c151796edcb4)
Tx side

![RX_LimeSDR](https://github.com/user-attachments/assets/128f2492-27c7-4bd0-b035-f406f5c85c87)
Rx side

Is there anything I'm doing incorrectly? 

Thanks

**Comment by duggabe** [comment, marked as answer]
Look at https://github.com/duggabe/gr-control/discussions/13
The same issues apply.

**Reply by ghost** [reply]
Thanks! I found that setting the HackRF's sample rate to 2MHz (instead of the 8MHz) and recalculating the respective interpolation and decimation prevented data loss in transmission.
